src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Update-AzureRmRecoveryServicesAsrProtectionDirection.md

```diff
@@ -51,6 +51,25 @@ Update-AzureRmRecoveryServicesAsrProtectionDirection [-VmmToVmm]
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
+### AzureToAzure
```

Comment:
Please add an example for these new parameter sets

Reply:
sure.